packages/react-components/react-button/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -225,6 +242,19 @@ const useRootStyles = makeStyles({
       ...shorthands.borderColor('transparent'),
       color: tokens.colorNeutralForeground2BrandPressed,
     },
+
+    '@media (forced-colors: active)': {
+      ':hover': {
+        backgroundColor: tokens.colorTransparentBackgroundHover,
+        ...shorthands.borderColor('transparent'),
+        color: 'Highlight',
+      },
+      ':hover:active': {
+        backgroundColor: tokens.colorTransparentBackgroundHover,
+        ...shorthands.borderColor('transparent'),
+        color: 'Highlight',
+      },
```

Comment:
re: the bundle size thing -- I wonder if a better approach would be to do a `const useHighContrastStyles = makeResetStyles( ...all shared HCM styles here )` and apply that className after the others